core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -186,6 +190,12 @@ private[spark] class CoarseMesosSchedulerBackend(
         .setValue(value)
         .build())
     }
+
+    // Set the ports to be pickedup by the executor
+    environment.addVariables(Environment.Variable.newBuilder()
+      .setName("SPARK_MESOS_PREASSIGNED_PORTS")
```

Comment:
I'd rather not introduce yet another env var.  Can't we just use the existing `SPARK_EXECUTOR_OPTS`?